runtime/jcl/common/sun_reflect_ConstantPool.c

```diff
@@ -907,6 +908,7 @@ registerJdkInternalReflectConstantPoolNatives(JNIEnv *env) {
 			(void *)&Java_sun_reflect_ConstantPool_getUTF8At0,
 		},
 	};
+	jint numNatives = sizeof(natives)/sizeof(JNINativeMethod);
```

Comment:
Pls declare c variables at the beginning of a scope.

Reply:
Should have looked at more context, this is the beginning.

Reply:
Yes, it is.